### PR TITLE
Kairosdb ttl support

### PR DESCRIPTION
**Pull request by jaroug**
Had a new option for the kairosdb ouput format.

TTL let you sets the Cassandra ttl for the data points.

### CODE REVIEW

File: src/collectd.conf.in [hunk 1]
```diff
@@ -1547,6 +1547,7 @@
 #		SSLVersion "TLSv1"
 #		Format "Command"
 #		Attribute "key" "value"     # only available for KAIROSDB format
+#		TTL 0   # data ttl, only available for KAIRODB format
```
Comment (rubenk) [inline src/collectd.conf.in]: KAIROSDB

File: src/collectd.conf.pod [hunk 2]
```diff
@@ -8778,6 +8778,14 @@ each metric being sent out.
 
 You can add multiple B<Attribute>.
 
+=item B<TTL> I<Int>
+
+Only available for KAIROSDB output format.
```
Comment (rubenk) [inline src/collectd.conf.pod]: the KAIROSDB output format

Comment (rubenk) [inline src/collectd.conf.pod]: Please fix this one too, then I'll merge.

Comment (jaroug) [inline src/collectd.conf.pod]: Hi, actually I fixed them both in one commit https://github.com/collectd/collectd/pull/2252/commits/4381f2ebb2edafb4b17aac9d1fc78a57f03b6913
Sorry if it's not a best pratice.

Comment (rubenk) [inline src/collectd.conf.pod]: No you didn't. you fixed the comment after "Attribute", but not after "TTL"

Comment (jaroug) [inline src/collectd.conf.pod]: just saw my mistake, my bad :)

Comment (rubenk) [inline src/collectd.conf.pod]: no problem :)

**Pull request merged**